wiki/Tournaments/SOLMT/2019/en.md

```diff
@@ -0,0 +1,341 @@
+| Top (#49 - #64) | ![][flag_NZ] [Blujae](https://osu.ppy.sh/users/10613885), ![][flag_RU] [TruthIsNear](https://osu.ppy.sh/users/11191093), ![][flag_MX] [YukariYakumo](https://osu.ppy.sh/users/10595982), ![][flag_ID] [StyxFearNite](https://osu.ppy.sh/users/8390933), ![][flag_ES] [Nikolayio](https://osu.ppy.sh/users/11279465), ![][flag_KR] [VarvalianFanboy](https://osu.ppy.sh/users/13211236), ![][flag_ru] [AsyaKys](https://osu.ppy.sh/users/12096335), ![][flag_US] [Dilly_P](https://osu.ppy.sh/users/12907431), ![][flag_MX] [Kenrit-San](https://osu.ppy.sh/users/13054150), ![][flag_UK] [zHotRoD](https://osu.ppy.sh/users/10441414), ![][flag_AU] [Crumpetfiddler](https://osu.ppy.sh/users/3518705), ![][flag_RU] [artslay](https://osu.ppy.sh/users/7758301), ![][flag_DO] [Kaito-kun](https://osu.ppy.sh/users/4715184), ![][flag_TH] [monkerzaza](https://osu.ppy.sh/users/11025701), ![][flag_ID] [[ Albedo ]](https://osu.ppy.sh/users/14584195), ![][flag_US] [Lights](https://osu.ppy.sh/users/9380939) | 
+
+### Groups
+
+| Group | Top | High | Mid | Low |
+| :-: | --- | --- | --- | --- |
+| **A** |![][flag_AU] \[Crz\]Lochie | ![][flag_KR] NKEYBOARD | ![][flag_BR] Halo- | ![][flag_TH] monkerzaza |  
+| **B** |![][flag_US] Unholy |![][flag_RU] Lerck |![][flag_MX] HéctorAF01 |![][flag_MX] YukariYakumo |
+| **C** |![][flag_PH] bojii |![][flag_RU] AngeloLagusa |![][flag_FR] DragonSonic |![][flag_UK] zHotRoD |
+| **D** |![][flag_TH] -\[DaNieL_TH]\-|![][flag_US] Playboi Carti |![][flag_PH] Shiyui- |![][flag_RU] TruthIsNear |
+| **E** |![][flag_PH] realqarp |![][flag_TW] AW ZERO |![][flag_US] yahweh |![][flag_RU] artslay |
+| **F** |![][flag_US] Logan636 |![][flag_MX] -Hellscythe- |![][flag_RU] FoXCHANNEL |![][flag_DO] Kaito-kun |
+| **G** |![][flag_US] HeZeuS |![][flag_ID] Proxae |![][flag_AR] jLuyalb |![][flag_ID] StyxFearNite |
+| **H** |![][flag_BR] SillyFangirl |![][flag_AR] ByDarking |![][flag_BR] Gabi Kamui |![][flag_ID] \[ Albedo \] |
+| **I** |![][flag_US] -Electro- |![][flag_FI] Strachy |![][flag_VN] baoketban123 |![][flag_MX] Kenrit-San |
+| **J** |![][flag_PH] Swarmsii |![][flag_PH] Cohue |![][flag_AR]Mateocristaldo |![][flag_KR] VarvalianFanboy |
+| **K** |![][flag_VN] MashedPotato |![][flag_BR] Kyut |![][flag_US] Jellyblob |![][flag_AU] Crumpetfiddler |
+| **L** |![][flag_MY] cheewee10 |![][flag_US] Trainer- |![][flag_DO] Antalf |![][flag_ES] Nikolayio |
+| **M** |![][flag_BR] andreymc |![][flag_RU] c1e4r |![][flag_FR] Ping36 |![][flag_RU] AsyaKys |
+| **N** |![][flag_US] stupud man |![][flag_FI] Camopoltergeist |![][flag_EC] Reo09 |![][flag_NZ] Blujae |
+| **O** |![][flag_US] dhn |![][flag_RU] N1ckn1ght |![][flag_CO] Cansta |![][flag_US] Lights  |
+| **P** |![][flag_PE] DaZeRoS |![][flag_CA] loafusofbread |![][flag_LV] CrayZigg |![][flag_US] Dilly_P |
+
+---
```

Comment:
You can delete this section (Line 93 - Line 106), and apply it again when the tournament is over.